vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -146,7 +145,10 @@ type OpenStack struct {
 	// Options for reading config data from a secret
 	secretName      string
 	secretNamespace string
+	// kubeconfigPath option is deprecated and shouldn't be used
 	kubeconfigPath  string
```

Comment:
isn't there some struct you marshal into? I guess it is not this one.

Reply:
done

Reply:
Is this var still used. Can we reven delete it?

Reply:
I think we shouldn't because it will break the installer, which sets the value. If it tries to set a non-existent value, the installation will fail.

Reply:
I think we can merge this patch, then remove setting of the value from the installer
https://github.com/openshift/installer/blob/master/pkg/asset/manifests/openstack/cloudproviderconfig.go#L33 and, if necessary, remove kubeconfigPath from here with a separate patch.

Reply:
I don't mean the config struct var, but this one (which is copied over from the former)

Reply:
ah, you're right. for sure we can remove it